collect_app/src/main/java/org/odk/collect/android/fragments/FormManagerList.java

```diff
@@ -203,8 +181,9 @@ public void onListItemClick(ListView l, View v, int position, long rowId) {
     @Override
     public void syncComplete(String result) {
         Timber.i("Disk scan complete");
```

Comment:
How about making sure each `syncComplete` implementation has this logging just in case an issue needs to be traced.

Reply:
Okay